views/helpers.go

```diff
@@ -0,0 +1,83 @@
+package views
+
+import (
+	"github.com/gorilla/sessions"
+	"github.com/ystv/web-auth/user"
+)
+
+type (
+	// Context is a struct that is applied to the templates.
+	Context struct {
+		Message  string
+		MsgType  string
+		Version  string
+		Callback string
```

Comment:
Could you add some comments explaining what these are for?

Reply:
I have added comments